src/renderer/stores/notebooks.js

```diff
@@ -57,6 +73,20 @@ var Notebooks = Backbone.Collection.extend({
     this.set(notebooks);
   },
 
+  handleCheckDeleteNotesReply: function(deleteStatus, id) {
+    this.destroyNotebook(id);
+    if (deleteStatus) {
+      console.log('deleting notes');
+      actionCreator.destroyNotesById(id);
+    } else {
+      actionCreator.resetNotebooks(id);
```

Comment:
we should be using actionCreator to dispatch the resetNotebooks action, and then catch/handle it in this store --> as we are doing with all the other actions in `dispatchCallback`